.github/ISSUE_TEMPLATE/release-template-new.md

```
@@ -0,0 +1,69 @@
+## Release process
+
+**1. Update version numbers and changelog**
+- [ ] [Run the Bump wp-parsely version](https://github.com/Parsely/wp-parsely/actions/workflows/bump-version.yml) GitHub Action to update the version numbers in the plugin files. Use the branch you want to release from (usually `develop`).
+- [ ] Verify that the generated PR looks correct. You can amend it with new commits if needed.
+- [ ] Merge the PR into the target branch (usually `develop`).
+
+**2. Merge develop into trunk**
+- [ ] [Create a PR](https://github.com/Parsely/wp-parsely/compare/trunk...develop?quick_pull=1&title=Release+wp-parsely+x.y.z&body=This+PR+merges+the+`develop`+branch+into+the+`trunk`+branch+in+order+to+release+wp-parsely+x.y.z.) that merges the target branch (usually `develop`) into `trunk`, named _Release wp-parsely x.y.z_.
+- [ ] Merge the PR into `trunk`.
+
+**3. Create Release and Deploy to WordPress.org**
```

Markdownlint

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)